src/limestone-api.ts

```diff
@@ -187,8 +186,9 @@ export default class LimestoneApi {
     const provider = _.defaultTo(opts.provider, this.defaultProvider);
```

Comment:
What do you this about generalising the step of defaulting opts to "this"? Otherwise we will need to add new line for every property that could be overwriten in getPrice opts.